### PR TITLE
Added Header and Blank Home Page

### PR DESCRIPTION
**Pull request by chintannp**
image

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+
+export default function Home() {
+    return (
+        <View style={styles.container}>
+            <Text >Todo App</Text>
```
Comment (calebpollman) [inline src/Home.js]: Nit: empty white space in the opening `Text` tag. Maybe we should add `prettier` to the project to clean these up, thoughts?

File: App.js [hunk 2]
```diff
@@ -1,19 +1,24 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify'
+import awsconfig from './src/aws-exports'
+import Home from './src/Home';
+import Header from './src/Header';
 
+Amplify.configure(awsconfig)
```
Comment (calebpollman) [inline App.js:9]: Nit: add a line break between this line and the `App` declaration

File: App.js [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Header />
+      <Home />
       <StatusBar style="auto" />
```
Comment (calebpollman) [inline App.js]: Should `StatusBar` be above the other components?

File: App.js [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
     </View>
   );
 }
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```
Comment (calebpollman) [inline App.js:16]: Why are you removing the `flex` property?

File: package.json [hunk 5]
```diff
@@ -8,6 +8,9 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/netinfo": "^6.0.2",
```
Comment (calebpollman) [inline package.json]: I don't see `@react-native-community/netinfo` or `aws-amplify-react-native` being used in this PR? I know we will need them eventually but as a best practice we should wait to add deps until they are needed

File: src/Header.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+export default function Header() {
+    return (
+        <View style={styles.headerContainer}>
+            <Text style={styles.headerTitle}>Todo App</Text>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    headerContainer: {
```
Comment (calebpollman) [inline src/Header.js]: We can go ahead and drop `header` here and just call this `container`

File: src/Header.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+        position: "relative",
```
Comment (calebpollman) [inline src/Header.js]: Is this property needed?

File: src/Header.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+        top: 0,
+        width: "100%",
+        paddingVertical: 30,
+        backgroundColor: '#4696ec',
+    },
+    headerTitle: {
```
Comment (calebpollman) [inline src/Header.js]: Similar to the above comment, we can drop `header` from this key

**Review by calebpollman — COMMENTED** [review]
Thanks for doing this! Can you please add a screen shot so i can get an idea of what the components look like?

File: App.js (hunk 2 above)
Comment (chintannp) [reply App.js:9]: Got it.

File: App.js (hunk 3 above)
Comment (chintannp) [reply App.js]: Nice Catch!. I'll update that.

File: App.js (hunk 4 above)
Comment (chintannp) [reply App.js:16]: I will be adding a ScrollView inside the Home Component in next commit. So, flex property around a ScrollView doesn't make sense.

File: package.json (hunk 5 above)
Comment (chintannp) [reply package.json]: That makes sense. I'll update it.

File: src/Header.js (hunk 6 above)
Comment (chintannp) [reply src/Header.js]: ok. will update that.

File: src/Header.js (hunk 7 above)
Comment (chintannp) [reply src/Header.js]: You are right. Its not needed.

File: src/Home.js (hunk 1 above)
Comment (chintannp) [reply src/Home.js]: I already installed it earlier. It was showing up some errors when I tried to format the code. Restarting the VSCode fixed it.

File: App.js (hunk 4 above)
Comment (calebpollman) [reply App.js:16]: Might be missing something, but ideally the containing `View` of an `App` should fill the available space of the device screen

Comment (chintannp) [reply App.js:16]: Thanks Caleb, I have added flex to the root Component and justified the content to `flex-start`.

Comment (calebpollman) [reply App.js:16]: **Maybe** we can just remove the `alignItems` and `justifyContent` properties altogether?

Comment (chintannp) [reply App.js:16]: I have pushed the new commit. It doesn't affect the styling at the moment.

**Review by calebpollman — APPROVED** [review]

**Pull request merged**